.github/dependabot.yml

```
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
+    directories:
+      - "**/*"
+    schedule:
+      interval: daily
```

```suggestion
      interval: "weekly"
```
Does the scheduling interval need to be set to daily? Is it too frequent? I think a weekly run is enough.

---

I don't think it makes much of a difference, with weekly it's more likely that you'll get a bunch of PRs all at once and reach the limit of open PRs

---

Which "limit of open PRs"?

klog does not have external Go dependencies. I am unsure what kind of PRs dependabot will create here and how useful they will be. I suppose we can try it out...

/lgtm
/assign @dims

For approval.

---

@pohly https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit

by default dependabot only opens 5 PRs for every package-ecosystem at a time